utils/sprites.py

```diff
@@ -251,7 +251,7 @@ def set_layer(self, layer=None):
     def set_label(self, new_label, i=0):
         ''' Set the label drawn on the sprite '''
         self._extend_labels_array(i)
-        if type(new_label) is str or type(new_label) is unicode:
+        if type(new_label) is str or type(new_label) is str:
```

Comment:
This line checks the same thing twice.  It is like `if False or False`, or `if True or True`.

Reply:
Also, other activities that contain sprites.py use isinstance instead of type; did you compare against them at all?